stdlib/importlib/resources/__init__.pyi

```diff
@@ -10,28 +11,45 @@ from typing_extensions import TypeAlias
 if sys.version_info >= (3, 9):
     from importlib.abc import Traversable
 
-__all__ = ["Package", "Resource", "contents", "is_resource", "open_binary", "open_text", "path", "read_binary", "read_text"]
+__all__ = ["Package", "contents", "is_resource", "open_binary", "open_text", "path", "read_binary", "read_text"]
 
 if sys.version_info >= (3, 9):
     __all__ += ["as_file", "files"]
 
 if sys.version_info >= (3, 10):
     __all__ += ["ResourceReader"]
 
+if sys.version_info >= (3, 13):
+    __all__ += ["Anchor"]
+else:
+    __all__ += ["Resource"]
+
 Package: TypeAlias = str | ModuleType
 
+if sys.version_info >= (3, 12):
```

Comment:
Added in 3.12, but not in `__all__` in 3.13. It seems they wanted to backport this, but it never happened: https://github.com/python/cpython/pull/113801

Should probably create a cpython issue here.